library/src/main/java/com/novoda/downloadmanager/NetworkFileDownloader.java

```diff
@@ -30,7 +30,7 @@ public void startDownloading(String url, FileSize fileSize, Callback callback) {
             processResponse(callback, response, responseCode, url);
         } catch (IOException e) {
             Logger.e(e, "Exception with http request");
-            callback.onError(e.getMessage());
+            callback.onError(FileDownloadError.createFrom(url, e.getMessage(), -1));
```

Comment:
what does -1 mean?

Reply:
Nothing at the moment, this will be iterated on before this is released. At the moment it probably means `UNEXPECTED` I guess.

Reply:
would be good to name it 👍

Reply:
d99707fa4fb0d34c3a1e7de7c740f8a4d8e0b57f